### ISSUE TITLE
description of CLI option "-n" is misleading

### ISSUE THREAD

**Issue by weinand** [Microsoft/vscode#42503]
From a Gitter user:

The description for the "-n" CLI option has this description:
```
-n, --new-window                          Force a new instance of Code.
```

This is misleading (or even wrong) because the options only forces a "new window".

On the other hand the CLI option "--user-data-dir" seems to create a new instance of VS Code, but doesn't say so in its description:
```
Specifies the directory that user data is kept in, useful when running as root.
```

We should improve the descriptions.

**Comment by bpasero** [comment]
And on top of that some users confuse this with a way of opening the same folder in multiple windows.

**Comment by ymolists** [comment]
@weinand thank you for filing this !

Here is how i explained it in the gitter channel:
I am running vscode in linux and i have been trying this for a while with no sucess ... i usually run vscode on my desktop. but sometimes i also login via vnc to my desktop from my laptop. For some reason vscode refuses categorically to run two instances one in vnc and one in the desktop. So i have to exit one to start the other. Most likely because when i pass -n vscode finds the currently running process and just opens a new window in that display ! ... For example if i try to run it from the vnc session while the desktop is running i will see a new window coming up in the desktop .. which is clearly not what i want

So basically all i want here is a way to start two instances of vscode windows but with seprate user-dir. Tell me how/if this is supported please

**Comment by bpasero** [comment]
I changed the settings to this:
`-n`: Force to open a new window.
`--user-data-dir`: Specifies the directory that user data is kept in. Can be used to open two distinct instances of Code.

**Comment by ymolists** [comment]
This is still misleading in my opinion because vscode does not open a new instance. I filed [ a new bug](https://github.com/Microsoft/vscode/issues/42514) for that behaviour to be corrected

**Comment by weinand** [comment]
@bpasero the `--user-data-dir` description now says  "Can be used to open two distinct instances of Code." 
I do not understand why this option can only be used for **two** instances of VS Code. 
If there is no such limitation, I suggest to drop the word "two".

Sorry, but I did not spot "two" in your comment above...

**Comment by bpasero** [comment]
@weinand yeah good point, I changed it to: "...can be used to open multiple distinct..."
